src/includes/performance-monitoring/configuration/vue.mdx

```diff
@@ -1,49 +1,54 @@
-
 **Vue.js**
 
 To get started with performance monitoring with Vue.js, first install these packages:
 
 ```bash
 # Using yarn
-$ yarn add @sentry/browser @sentry/integrations @sentry/apm
+$ yarn add @sentry/browser @sentry/integrations @sentry/tracing
 
 # Using npm
-$ npm install @sentry/browser @sentry/integrations @sentry/apm
+$ npm install @sentry/browser @sentry/integrations @sentry/tracing
 ```
 
+<Alert level="warning" title="Note"><markdown>
+
+We switched our package name from `@sentry/apm` to `@sentry/tracing`. If you were previously using `@sentry/apm`, please see [Switching to the tracing package](/product/performance/apm-to-tracing/)
+
+</markdown></Alert>
+
 The Vue Tracing Integration allows you to track rendering performance during an initial application load.
 
 Sentry injects a handler inside Vue's `beforeCreate` mixin, providing access to a Vue component during its life cycle stages.
-When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
-and create a new activity named `Vue Application Render`. Once the activity has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the activity as completed.
+When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM BrowserTracing integration,
+and create a new span named `Vue Application Render`. Once the span has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the span as finished.
```

Comment:
Spans don't have timeouts, do they? Is this actually a transaction?

Reply:
Ah this is based on `idleTransaction`. Maybe it is better just to remove this paragraph.